node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/HostFlavorUpgrader.java

```diff
@@ -80,6 +80,8 @@ private double upgradeHostFlavor(NodeList allNodes) {
                 return 1.0;
             } catch (NodeAllocationException e) {
                // Fine, no capacity for upgrade
+                log.log(Level.WARNING, "Failed to redeploy " + node.allocation().get().owner() + " to upgrade flavor (" +
```

Comment:
I don't think we should warn here? We'll emit a warning each time we try but can't upgrade to a newer flavor ...

Reply:
Yes, but I need it for debugging an issue. I can remove it later.